controller/controller.go

```diff
@@ -154,18 +159,39 @@ func (c *Controller) RunOnce(ctx context.Context) error {
 	return nil
 }
 
-// Run runs RunOnce in a loop with a delay until stopChan receives a value.
-func (c *Controller) Run(ctx context.Context, stopChan <-chan struct{}) {
-	ticker := time.NewTicker(c.Interval)
+// MIN_INTERVAL is used as window for batching events
+const MIN_INTERVAL = 5 * time.Second
```

Comment:
thoughts on making configurable via `external-dns` CLI flags?

Reply:
I'd prefer to do this in separate PR, this PR is meant to fix current issues

Reply:
👍